docs/source/maintenance/metrics.rst

```diff
@@ -0,0 +1,39 @@
+=======
+Metrics
+=======
+
+To conveniently and thoroughly monitor the performance of the network, you can now use metrics.
+It is numeric data collected over time about your Iroha network.
+You can then analyse the data to make your project even more efficient!
+
+How to use metrics?
+===================
+
+To use metrics, you need to add it to your `Iroha configuration <../configure/index.html#deployment-specific-parameters>`_ and use Iroha version that is newer than 1.2.1.
+
+Then, you can simply use the ip address to access the data from the running Iroha instance.
+
+Here is an example:
+
+.. code-block:: shell
+
+  > curl http://127.0.0.1:8080/metrics
+
+  	will give you results like: 
+
+
+	# HELP blocks_height Total number of blocks in chain
```

Comment:
please remove redundant identation.

hint: you can edit it on GitHub with instant preview